src/setup/globals.js

```diff
@@ -30,4 +30,9 @@ window.ProcessMaker = {
 
     window.ProcessMaker.EventBus.$emit('alert', window.ProcessMaker.navbar.alerts);
   },
+  // user: {
+  //   timezone: 'America/Recife',
+  //   lane: 'en',
+  //   datetime_format: 'MM/DD/YYYY HH:mm',
+  // },
```

Comment:
I added this for testing but left it commented out, as this changes the displayed format and thus will cause tests to fail.

Should I leave it like this for now? I think it would be nice to add it to a test that confirms that things work in the local timezone and in some other user defined timezone.

Reply:
Seems like an integration test. Wonder how that would work here. @JCook21 ?

Reply:
Since this info exists on the window object, I think we can change that object from the test. The XML generated should be the same, but that formatting the user sees in the input will be different.

Reply:
I'll leave this for now and pull it out into a test in a separate PR.

Reply:
Can it be deleted? I don't think leaving commented code in is ever a good idea...